File: trunk/quests/qeynos2/Leanon_Ruksey.pl
sub EVENT_SAY { 
if($text=~/Hail/i){
quest::say("Ah..  good day. friend.  My name's Leanon Ruksey. loyal member of the League of Antonican Bards..  Qeynos Chapter. of course.  It's part of my job to keep up with the goings-on of the city. who's coming and going and all that.  So we usually keep a man here and one at the docks.  Anyway. nice chatting with you..  be careful out there!"); }
}

sub EVENT_ITEM {
  if (plugin::check_handin(\%itemcount, 18020 =>1 )) {
    quest::say("Ah, hello there, $name. It's such a fine day out, wouldn't you say? Ah, reports are due in. Very well, here you go. Say hello to the fellows back at the Wind Spirit's Song for me. May you have many safe travels, my friend.");
    quest::summonitem(18024);
    quest::faction( 192,2);
   quest::faction( 184,2);
   quest::faction( 135,2);
   quest::faction( 273,-2);
   quest::faction( 207,-2);
   quest::exp(200);
  }
  plugin::return_items(\%itemcount);
}
#END of FILE Zone:qeynos2  ID:2012 -- Leanon_Ruksey 

File: trunk/quests/qeynos2/Leanon_Ruksey.lua
function event_say(e)
	if(e.message:findi("hail")) then
		e.self:Say("Ah..  good day. friend.  My name's Leanon Ruksey. loyal member of the League of Antonican Bards..  Qeynos Chapter. of course.  It's part of my job to keep up with the goings-on of the city. who's coming and going and all that.  So we usually keep a man here and one at the docks.  Anyway. nice chatting with you..  be careful out there!");
	end
end

function event_trade(e)
	local item_lib = require("items");

	if(item_lib.check_turn_in(e.trade, {item1 = 18020})) then
		e.self:Say(string.format("Ah, hello there, %s. It's such a fine day out, wouldn't you say? Ah, reports are due in. Very well, here you go. Say hello to the fellows back at the Wind Spirit's Song for me. May you have many safe travels, my friend.",e.other:GetName()));
		e.other:SummonItem(18024);
		e.other:Ding();
		e.other:Faction(192,2,0);
		e.other:Faction(184,2,0);
		e.other:Faction(135,2,0);
		e.other:Faction(273,-2,0);
		e.other:Faction(207,-2,0);
		e.other:AddEXP(200);
	end
	item_lib.return_items(e.self, e.other, e.trade)
end
